.github/workflows/pull-compliance.yml

```diff
@@ -63,15 +57,11 @@ jobs:
           GOPROXY: https://goproxy.io # proxy.golang.org is blocked in China, this proxy is not
           GOSUMDB: sum.golang.org
           TAGS: bindata gogit sqlite sqlite_unlock_notify
-      - name: checks backend
-        run: make --always-make checks-backend # ensure the 'go-licenses' make target runs
```

Comment:
Deleted by accident?

Reply:
Forgot to mention but this is intentional because the same command runs in the following step again.

Reply:
Updated description.